File: LogicadeProgramacao/Estrutura Sequencial/2. Java/TempoDownload.java
import java.util.Scanner;

public class TempoDownload {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double tamanhoArquivo, velocidadeLink, tempoDownload;

        System.out.print("Digite o tamanho do arquivo em MB: ");
        tamanhoArquivo = entrada.nextDouble();

        System.out.print("Digite a velocidade do link de internet em Mbps: ");
        velocidadeLink = entrada.nextDouble();

        tempoDownload = (tamanhoArquivo * 8) / velocidadeLink / 60;

        System.out.printf("O tempo aproximado de download é de %.2f minutos.", tempoDownload);

        entrada.close();
    }
}

File: LogicadeProgramacao/Estrutura Sequencial/3. Python/TempoDownload.py
tamanho_arquivo = float(input("Digite o tamanho do arquivo em MB: "))
velocidade_link = float(input("Digite a velocidade do link de internet em Mbps: "))

tempo_download = (tamanho_arquivo * 8) / velocidade_link / 60

print(f"O tempo aproximado de download é de {tempo_download:.2f} minutos.")
